pynbody/snapshot/__init__.py

```diff
@@ -35,8 +35,105 @@
 
 logger = logging.getLogger('pynbody.snapshot')
 
+class UnitFullContainer:
```

Comment:
I like this solution but could it be in its own file? The `snapshot/__init__.py` is already far too long and diverse.

Also I wonder whether the name is clear? Perhaps something like `ContainerWithPhysicalUnitsOption`, while clunky, is clearer?

Reply:
Done